### PR TITLE
Service Fabric - added intros for heading sections

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-fabric.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-18/docs-ref-services/service-fabric.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180131383884-118%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/service-fabric.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ Azure Service Fabric is a distributed systems platform that makes it easy to pac
 
 Learn more about [Azure Service Fabric](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-overview).
 
-## Management Package
+## Management package
 
 ### Install npm module
 
-Use npm to install the Azure Service Fabric module for Node.js
+Install the Azure Service Fabric npm module
```
Comment (clarkio) [inline docs-ref-services/service-fabric.md]: Following the template, this should be `npm package` instead.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-fabric.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-18/docs-ref-services/service-fabric.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180357140779-118%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/service-fabric.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-18/docs-ref-services/service-fabric.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180400430170-118%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@johnpapa @clarkio merged

**Pull request merged**